src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,3 +1,34 @@
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+import './PostInfo.scss';
+
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({
+  post: {
+    title,
+    user,
+    body,
+    comments,
+  },
+}) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {title}
+      </h3>
+
+      <UserInfo user={user} />
+    </div>
+
+    <p className="PostInfo__body">
+      {body}
+    </p>
+
+    <CommentList comments={comments} />
```

Comment:
```suggestion
    <CommentList comments={comments} />
```
here, if no comments provided, you should print message "No comments yet"

Reply:
Nice comment, fixed!)